src/shared/telemetry/telemetryEvent.ts

```diff
@@ -10,6 +10,14 @@ import { MetadataEntry, MetricDatum, Unit } from './clienttelemetry'
 const NAME_ILLEGAL_CHARS_REGEX = new RegExp('[^\\w+-.:]', 'g')
 const REMOVE_UNDERSCORES_REGEX = new RegExp('_', 'g')
 
+export const METADATA_FIELD_NAME = {
+    RESULT: 'result',
+    DURATION: 'duration',
+}
+
+export const METADATA_RESULT_PASS = 'Succeeded'
```

Comment:
Would it make sense for this to be an enum?

Reply:
I considered enum, but ultimately we want to emit a string, so I opted for the simple approach